### PR TITLE
feat: add no-share option

### LINKED ISSUE #77: Request: only printing "View eval results" if sharable URL is present
I have `sharing: false` in my `promptfooconfig.yaml`.

The request is to only print "View eval results" ([source](https://github.com/promptfoo/promptfoo-action/blob/v1/src/main.ts#L89)) if `sharing` is enabled.

### PR DESCRIPTION
**Pull request by mocyuto**
Adding no share option.
If there are multiple promptfooconfig in a repositry, I want to block sharing at actions setting.

And also resolve #77 

This is screenshot.
スクリーンショット 2023-12-07 16 27 00

### CODE REVIEW

**Comment by mocyuto** [comment]
when set this option, output format cannot resolve..
```
/home/runner/.npm/_npx/1ddbeaccabfe00a4/node_modules/promptfoo/dist/src/util.js:220
        throw new Error(`Unsupported output file format ${outputExtension}, please use csv, txt, json, yaml, yml, html.`);
              ^

Error: Unsupported output file format , please use csv, txt, json, yaml, yml, html.
    at writeOutput (/home/runner/.npm/_npx/1ddbeaccabfe00a4/node_modules/promptfoo/dist/src/util.js:220:15)
    at writeMultipleOutputs (/home/runner/.npm/_npx/1ddbeaccabfe00a4/node_modules/promptfoo/dist/src/util.js:182:9)
    at Command.<anonymous> (/home/runner/.npm/_npx/1ddbeaccabfe00a4/node_modules/promptfoo/dist/src/main.js:339:49)
```

File: src/main.ts [hunk 1]
```diff
@@ -83,13 +89,20 @@ export async function run(): Promise<void> {
...
-      const body = `⚠️ LLM prompt was modified in these files: ${modifiedFiles}
+      let body = `⚠️ LLM prompt was modified in these files: ${modifiedFiles}
 
 | Success | Failure |
 |---------|---------|
 | ${output.results.stats.successes}      | ${output.results.stats.failures}       |
 
-**» [View eval results](${output.shareableUrl}) «**`;
+`;
+      if (output.sharableUrl) {
```
Comment (typpo) [inline src/main.ts:99]: Noticed the `sharableUrl` typo here. I've properly typed the output on `main`, so if we rebase and fix this error, we should be good to merge.

Comment (mocyuto) [reply src/main.ts]: I fixed it!

**Comment by typpo** [comment]
Thank you @mocyuto!

**Pull request merged**